content/en/learn/MakerDAO/maker-foundation.mdx

```diff
@@ -10,78 +10,137 @@ description: "Learn about MakerDAO."
 
 ### Who Started MakerDAO and When?
 
+<Box sx={{ px: 4 }}>
+
 The idea of MakerDAO became public on March 26th, 2015 with Rune Christensen’s post, [Introducing eDollar, the ultimate stablecoin built on Ethereum](reddit.com/r/ethereum/comments/30f98i/introducing_edollar_the_ultimate_stablecoin_built/), on the r/ethereum subreddit.
 
-### Why did the founders call it \_Maker_DAO?
+</Box>
+
+
+### Why did the founders call it MakerDAO?
+
+<Box sx={{ px: 4 }}>
 
-The term "Maker" is from the term Market Maker. This was driven in part based upon [BitUSD's](https://web.archive.org/web/20200426202839/https://bitshares.org/technology/price-stable-cryptocurrencies/) inability to attract market makers, which resulted in its lack of robust liquidity. One original idea was to distribute MKR specifically to crypto market makers, which we later started calling keepers, to ensure they were incentivized in the system and would help bootstrap its liquidity.
+The term **"Maker"** is from the term Market Maker. This was driven in part based upon [BitUSD's](https://web.archive.org/web/20200426202839/https://bitshares.org/technology/price-stable-cryptocurrencies/) inability to attract market makers, which resulted in its lack of robust liquidity. One original idea was to distribute MKR specifically to crypto market makers, which we later started calling keepers, to ensure they were incentivized in the system and would help bootstrap its liquidity.
```

Comment:
i think removing the we here is workable. "which MakerDAO starting calling keepers"

Reply:
what do you think @annaalexakr ?